swtr/server.py

```diff
@@ -63,15 +79,15 @@ def index():
     #print auth_tok
     payload = {'what': 'img-anno',
                'access_token': auth_tok['access_token']}
-    req = requests.get(config.swtstoreURL + '/api/sweets/q', params=payload)
+    req = requests.get(current_app.config.get('SWTSTORE_URL', 'SWTSTORE_URL') + '/api/sweets/q', params=payload)
     sweets = req.json()
     return render_template('index.html', access_token=auth_tok['access_token'],
                            refresh_token=auth_tok['refresh_token'],
-                           config=config, url=request.args.get('where'),
-                           bookmark=quote_plus(config.app_url), sweets=sweets)
+                           config=current_app.config, url=request.args.get('where'),
+                           bookmark=quote_plus(current_app.config.APP_URL), sweets=sweets)
```

Comment:
@romanchyla When I try to run this application, I get this error:

 `"AttributeError: 'Config' object has no attribute 'APP_URL'" `

I believe we'd have to change all occurrences of `current_app.config.SOME_KEY` to `current_app.config.get('SOME_KEY')`.

When I tried to fix that, I keep getting `BuildError: ('annotate_webpage', {}, None)`  which is weird. I don't understand why.